supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -143,6 +143,18 @@ The practice of having both modes together is often referred to as "hybrid," "ag
 
 *See also*:
 
+[discrete]
+[[blacklist]]
+==== blacklist (noun,verb,adjective)
+*Description*: A blacklist is a list of items that are disallowed or denied. It is an example of problematic language. Use "blocklist" or "denylist" instead.
+
+*Use it*: no
+
+*Incorrect forms*: blacklist, black-list
+
+*See also*: allowlist, blocklist, denylist, passlist, whitelist
```

Comment:
My intention was to later introduce the correct terms, and all these should be linked together, I believe. Would that make sense?

Reply:
@jafiala Yes, if you have entries for these correct terms, then it definitely makes sense to link to them here. If you're going to add them in this same PR, then that's fine to leave them here for now. But if you want this PR merged before those entries are added, then you'll need to remove them before it can be merged.